doc/nrf/ug_nrf5340.rst

```diff
@@ -0,0 +1,155 @@
+
+The cores use Interprocessor Communication (IPC) to notify each other if data is available for the other core to pick up.
+The actual data exchange is handled by Open Asymmetric Multi Processing (OpenAMP).
+
+The IPC driver is implemented in `nrfx`_.
+`OpenAMP`_ is included as module in Zephyr.
+
+
+Available samples
+*****************
+
+nRF5340 samples consist of two separate images: one that runs on the network core and one that runs on the application core.
+
+Network sample
+==============
+
+.. For network communication, Zephyr provides the :ref:`zephyr:bluetooth-hci-openamp-sample` sample that implements a Bluetooth Low Energy controller.
+
+For network communication, Zephyr provides a sample that implements a Bluetooth Low Energy controller.
+
+This sample must be programmed to the network core to run any Bluetooth Low Energy samples on nRF5340.
+
+You might need to adjust the Kconfig configuration of this sample to make it compatible with the peer application.
+For example, :option:`zephyr:CONFIG_BT_MAX_CONN` must be equal to the maximum number of connections supported by the application sample.
+
+
+Application samples
+===================
+
+The |NCS| provides a series of :ref:`Bluetooth Low Energy samples <ble_samples>`, in addition to the :ref:`Bluetooth samples in Zephyr <zephyr:bluetooth-samples>`.
+Most of these samples should run on the nRF5340 PDK, but not all have been thoroughly tested.
+Samples that use non-standard features of the Bluetooth Low Energy controller, like the :ref:`ble_llpm` sample, are not supported.
+
+.. Some samples require configuration adjustments to the :ref:`zephyr:bluetooth-hci-openamp-sample` sample as described in the `Network sample`_ section.
+
+Some samples require configuration adjustments to the network sample as described in the `Network sample`_ section.
+
+.. TODO - kapi: what adjustments are needed to the throughput sample?
+
+These samples must be programmed to the application core, in the secure domain.
+
+
+Building and programming a sample
+*********************************
+
+You must program both the network sample and one of the supported :ref:`Bluetooth Low Energy samples <ble_samples>`.
+
+Build and program both samples separately by following the instructions in :ref:`gs_programming_ses`.
+Make sure to use ``nrf5340_dk_nrf5340_cpunet`` as board name when building the network sample, and ``nrf5340_dk_nrf5340_cpuapp`` when building the application sample.
+
+.. important::
+   When programming the samples from |SES|, you might get an error message stating that the target cannot be identified.
+   In this case, you can either modify your |SES| installation and projects to add support for programming nRF5340, or program the generated HEX files from the command line instead.
+
+   See the following sections for more information.
+
+
+Adding support for programming nRF5340 in |SES|
+===============================================
+
```

Comment:
You don't need this for building, only for programming. :(
But it's a temporary section anyway, so it'll be removed soon.

Reply:
"programming" is mentioned in the first paragraph, so it clarifies the section title. Alternatively, maybe just use "SES" instead of "|SES|" in the title to make it shorter?